src/test/app/DepositAuth_test.cpp

```diff
@@ -634,6 +653,55 @@ struct DepositPreauth_test : public beast::unit_test::suite
                 sendmax(XRP(10)),
                 ter(expect));
             env.close();
+
+            {
+                // becky setup depositpreauth with credentials
+                const char credType[] = "abcde";
+                Account const carol{"carol"};
+                env.fund(XRP(5000), carol);
+
+                TER const expect{
+                    supportsPreauth ? TER{tesSUCCESS} : TER{temDISABLED}};
+
+                env(deposit::authCredentials(becky, {{carol, credType}}),
+                    ter(expect));
+                env.close();
+
+                // gw accept credentials
+                env(credentials::create(gw, carol, credType), ter(expect));
+                env.close();
+                env(credentials::accept(gw, carol, credType), ter(expect));
+                env.close();
```

Comment:
These transactions should still succeed if the DepositPreauth amendment isn't enabled.

Reply:
Added additional check